test/onnx/test_onnx_opset.py

```diff
@@ -107,5 +107,29 @@ def test_maxpool(self):
         x = torch.randn(20, 16, 50)
         check_onnx_opsets_operator(module, x, ops, opset_versions=[10])
 
+    def test_dropout(self):
+        class MyModule(Module):
+            def __init__(self):
+                super(MyModule, self).__init__()
+                self.dropout = torch.nn.Dropout(0.5)
+
+            def forward(self, x):
+                return self.dropout(x)
+
+        x = torch.randn(1, 2, 3)
+
+        # we should only export the onnx Dropout op in training mode; test both modes
+
+        # test training mode
```

Comment:
I think the training proposal is still in progress. So in ONNX, the dropout operator still only support inference mode. We should error out and print out warning if it's in training mode. Tell user, you should call .eval before exporting to onnx.

Reply:
So for both opset 9 and 10, error out instead of exporting Dropout() ?

Reply:
could we maybe still export the model with Dropout in training mode, but add a warning to inform the user?

Reply:
the only difference between opset 7 and opset 10's dropout is that their mask type (2nd output) is different. So they both only support inference, no training... So please don't add support for training yet. (ONNX's training proposal is still under development)